### ISSUE TITLE
Some schema ref links are broken

### ISSUE THREAD

**Issue by gokerakc** [asyncapi/spec-json-schemas#493]
### Describe the bug you found in AsyncAPI Docs.

There are broken links in the [sns](https://github.com/asyncapi/spec-json-schemas/blob/master/bindings/sns/0.1.0/operation.json#L27) and [sqs](https://github.com/asyncapi/spec-json-schemas/blob/master/bindings/sqs/0.2.0/operation.json#L99) binding schemas. 

You can see some of the broken links below:
- http://asyncapi.com/bindings/sns/0.1.0/operation.json#/definitions/deliveryPolicy
- https://www.asyncapi.com/bindings/sns/0.1.0/operation.json#/definitions/identifier
- http://asyncapi.com/bindings/sns/0.1.0/operation.json#/definitions/consumer
- http://asyncapi.com/bindings/sqs/0.2.0/operation.json#/definitions/policy

It seems like there is a problem with the **/bindings** path.

### Important

Apart from those there are too many other files that use broken links, please check the following link:
- https://github.com/search?q=repo%3Aasyncapi%2Fspec-json-schemas%20http%3A%2F%2Fasyncapi.com%2Fbindings%2F&type=code

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gokerakc** [comment]
I've created a bug report for the [website](https://github.com/asyncapi/website) repo since the website links need to be fixed: https://github.com/asyncapi/website/issues/2750

**Comment by gokerakc** [comment]
I guess the ideal solution is to update the $ref to use the definition that already exists in the schema. For example, this [ref](https://github.com/asyncapi/spec-json-schemas/blob/master/bindings/sns/0.1.0/operation.json#L27) already defined in [here](https://github.com/asyncapi/spec-json-schemas/blob/master/bindings/sns/0.1.0/operation.json#L153)

**Comment by smoya** [comment]
Bindings are not being served **yet** via AsyncAPI website. Here is the followup issue https://github.com/asyncapi/website/issues/870
Once that is done, those will work.

**Comment by rico33631** [comment]
Hi @smoya, I would love to work on this issue.

I used grep -r "http://asyncapi.com/bindings/" . to identify all occurrences of the links. The issue seems to be due to incorrect references, like a typo in the filename (channoel.json instead of channel.json), causing the links to break, but even after fixing the typos, some referenced definitions  don’t exist in the respective JSON files. For the missing definitions, I’d like to confirm:

- Were they accidentally omitted and should be added?
- Or are the references outdated and should be removed/updated?

Thanks!
